actors/miner/tests/util.rs

```diff
@@ -1509,9 +1511,9 @@ impl ActorHarness {
         rt.call::<Actor>(
             Method::DeclareFaultsRecovered as u64,
             &RawBytes::serialize(params).unwrap(),
-        )
-        .unwrap();
+        )?;
```

Comment:
Instead of using the `?` you can bind to the output of call.  Then if its an error you can `rt.reset()` instead of `rt.verify()`.  If you don't do this you need to call rt.reset after every expect_abort call to this helper method which is easier to get wrong.

Reply:
Modified `declare_recoveries` to implement this logic.